branding/ovirt-brand.css

```diff
@@ -0,0 +1,134 @@
+/********************************************
```

Comment:
Original idea of `branding` directory was that it will contain branding specific styles. Colors, image urls and dimensions are clearly branding specific. However rules like

```
.obrand_ssoHeader {
    height: 41px;
    position: relative;
    top: 4px;
    /* background-color: #121212; */
}
```

contains declarations most probably common to all brandings - e.g. `height: 41px;`.

If this is intentionally chosen way to go since it is easy to create brandings like that by copy-pasting parts of engine code, I'm ok with that.

Otherwise please move declarations common to all brandings to `index-nomodules.css` or similar and let `branding` directory to contain only branding specific declarations - colors, image urls, image dimensions, etc. This will help us make sure that branding packages differs only intentional aspects.

Reply:
I think I understand what you are saying.

This particular style 'obrand_ssoHeader' is affected by the height of the top margin, which is 4px now. If you had a d/s brand that wanted to use only 2px, you'd set that here. So it is a branding-important class and should be here. Are you wanting the class split across two different files, where 'height' and 'position' would be in nomodules, and 'top' would be in here?

Reply:
Based on IRC discussion: let's keep this format to make it easy to copy styles form engine.